lib/src/view/broadcast/broadcast_boards_tab.dart

```diff
@@ -0,0 +1,296 @@
+        AsyncError(:final error) => Center(
+            child: Text(error.toString()),
+          ),
+        _ => const Shimmer(
+            child: ShimmerLoading(
+              isLoading: true,
+              child: BroadcastPreview(
+                roundId: BroadcastRoundId(''),
+                title: '',
+              ),
+            ),
+          ),
+      },
+    );
+  }
+}
+
+class BroadcastPreview extends StatelessWidget {
+  final BroadcastRoundId roundId;
+  final IList<BroadcastGame>? games;
+  final String title;
+
+  const BroadcastPreview({
+    required this.roundId,
+    this.games,
+    required this.title,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    // TODO uncomment when eval bar is ready
+    // final showEvaluationBar = ref.watch(
+    //   broadcastPreferencesProvider.select((value) => value.showEvaluationBar),
+    // );
+    // TODO remove when eval bar is ready
+    const showEvaluationBar = false;
+    const numberLoadingBoards = 12;
+    const boardSpacing = 10.0;
+    // height of the text based on the font size
+    // since the TextStyle is defined with an height at 1.0, this is the real height
+    // see: https://api.flutter.dev/flutter/painting/TextStyle/height.html
+    final textHeight = _kPlayerWidgetTextStyle.fontSize!;
+    final headerAndFooterHeight = textHeight + _kPlayerWidgetPadding.vertical;
+    final numberOfBoardsByRow = isTabletOrLarger(context) ? 4 : 2;
+    final screenWidth = MediaQuery.sizeOf(context).width;
+    final boardWithMaybeEvalBarWidth = (screenWidth -
+            Styles.horizontalBodyPadding.horizontal -
+            (numberOfBoardsByRow - 1) * boardSpacing) /
+        numberOfBoardsByRow;
+
+    return GridView.builder(
+      padding: Styles.bodyPadding,
+      itemCount: games == null ? numberLoadingBoards : games!.length,
+      gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
+        crossAxisCount: numberOfBoardsByRow,
+        crossAxisSpacing: boardSpacing,
+        mainAxisSpacing: boardSpacing,
+        mainAxisExtent: boardWithMaybeEvalBarWidth + 2 * headerAndFooterHeight,
+        childAspectRatio: 1 + evaluationBarAspectRatio,
+      ),
+      itemBuilder: (context, index) {
+        final boardSize = boardWithMaybeEvalBarWidth -
+            (showEvaluationBar
+                // TODO remove when eval bar is ready
+                // ignore: dead_code
+                ? evaluationBarAspectRatio * boardWithMaybeEvalBarWidth
+                : 0);
+
+        if (games == null) {
+          return BoardThumbnail.loading(
+            size: boardSize,
+            header: _PlayerWidgetLoading(width: boardWithMaybeEvalBarWidth),
+            footer: _PlayerWidgetLoading(width: boardWithMaybeEvalBarWidth),
+            showEvaluationBar: showEvaluationBar,
+          );
+        }
+
+        final game = games![index];
+        final playingSide = Setup.parseFen(game.fen).turn;
+
+        return BoardThumbnail(
+          onTap: () {
```

Comment:
Here I think you can pass an `animationDuration` to the `BoardThumbnail` to animate position change.